pkg/cmd/adm/restart_test.go

```diff
@@ -221,22 +395,31 @@ func newDeployment(namespacedName types.NamespacedName, replicas int32) *appsv1.
 		},
 		Spec: appsv1.DeploymentSpec{
 			Replicas: &replicas,
+			Selector: &metav1.LabelSelector{MatchLabels: map[string]string{"host": "controller"}},
```

Comment:
why do you set some "host" label when the function is called also for member operator? If you just want to simply reuse the same function, then use some dummy value there
```suggestion
			Selector: &metav1.LabelSelector{MatchLabels: map[string]string{"dummy-key": "controller"}},
```